src/macros.rs

```diff
@@ -150,3 +150,26 @@ macro_rules! wrap_pymodule {
         }
     };
 }
+
+#[cfg(not(PyPy))]
+/// Add the module to the initialization table in order to make embedded Python code to use it.
+/// Module name is the argument.
+///
+/// Use it before [`prepare_freethreaded_python`](crate::prepare_freethreaded_python) and
+/// leave feature `auto-initialize` off
+#[macro_export]
+macro_rules! append_to_inittab {
+    ($module:ident) => {
+        unsafe {
+            ::std::assert_eq!(
```

Comment:
Hmmm, looks like `assert_eq` is not hygienic until 1.50. @mejrs how do you feel if we just put `#[rust version::since(1.50)]` on this test and live with it? It's such a niche case that messying the implementation for the older compilers (to support only `no_implicit_prelude`) doesn't seem worth it.

Reply:
It can just be `if a != b { panic!()}` rather than an assert_eq (may need an allow to stop clippy from complaining about it though)